### PR TITLE
Add PR source validation to Jenkinsfile

### PR DESCRIPTION
**Pull request by jakedipity**
Semver: patch
Ref: LOG-7715

Signed-off-by: Jacob Hull <jacob@planethull.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -108,7 +125,7 @@ pipeline {
         versioner(
           token: "${GITHUB_PACKAGES_TOKEN}"
         , dry: true
-        , repo: repo
+        , repo: REPO
```
Comment (mdeltito) [inline Jenkinsfile:128]: there's one more of these, line 162

**Review by mdeltito — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (jakedipity) [reply Jenkinsfile:128]: fixed

**Review by mdeltito — DISMISSED** [review]

**Review by darinspivey — DISMISSED** [review]

**Comment by mdeltito** [comment]
@logdnabot test this

**Comment by jakedipity** [comment]
@logdnabot test this

**Comment by jakedipity** [comment]
There appears to be an issue with Jenkins status checks... one of the status checks is hanging and Jenkins is not reporting any secondary tasks running / started / stopped

**Comment by jakedipity** [comment]
@logdnabot test this

**Comment by jakedipity** [comment]
@logdnabot test this

**Review by mdeltito — APPROVED** [review]

**Review by esatterwhite — APPROVED** [review]

**Review by dm36 — APPROVED** [review]

**Pull request merged**